socs/agents/meinberg_syncbox/agent.py

```diff
@@ -61,6 +65,25 @@ def _extract_oid_field_and_value(get_result):
     if not isinstance(oid_value, (int, bytes, str)):
         oid_value = None
 
+    if (field_name.split('_')[0] in CONVERT_OIDS) and (oid_value is not None):
+        unit_multipliers = {
+            "s": 1000000,   # seconds to microseconds
+            "sec": 1000000,  # seconds to microseconds
+            "ms": 1000,     # milliseconds to microseconds
+            "us": 1,        # microseconds
+            "ns": 0.001     # nanoseconds to microseconds
```

Comment:
Small nit, but since they're mostly aligned already...

```suggestion
            "s": 1000000,    # seconds to microseconds
            "sec": 1000000,  # seconds to microseconds
            "ms": 1000,      # milliseconds to microseconds
            "us": 1,         # microseconds
            "ns": 0.001      # nanoseconds to microseconds
```